app/src/main/java/br/org/cesar/discordtime/stickysessions/data/remote/service/RemoteServiceFactory.java

```diff
@@ -18,13 +21,14 @@
     private static final int CACHE_SIZE = 10 * 1024 * 1024;
 
     public T makeRemoteService(Context context, String baseUrl, boolean isDebug,
-                               Class<T> serviceClass) {
-        OkHttpClient okHttpClient = makeOkHttpClient(context, makeLoggingInterceptor(isDebug));
+                               Class<T> serviceClass, INetworkWrapper networkWrapper) {
+        OkHttpClient okHttpClient =
+                makeOkHttpClient(context, makeCustomHttpInterceptor(isDebug, networkWrapper));
```

Comment:
That way makeLoggingInterceptor function is not used anymore. I would expect to decide for custom interceptor or not via dependency injection. Is this working for local flavor? I think app using local flavor shouldn't require internet connection, right?